docs/content/v0.20/packaging.md

```diff
@@ -47,19 +51,40 @@ The files in [pkg/generated](../../../pkg/generated) are generated by [k/code-ge
 
 ### Central container image
 
-Currently at `ghcr.io/kubestellar/kubestellar/kubestellar-operator:0.20.0-alpha.1`.
+KubeStellar has one container image, for what is called the
+KubeStellar operator. This operator installs the needed custom
+resource _definition_ objects if they are not already present, and is
+a controller-manager hosting the relevant central controllers.
```

Comment:
It's not clear what are those "relevant central controllers" . So I think we should either explain + examples , or remove it.

Reply:
I plan to update this later with a reference into the architecture document, which I expect will be filled out with more content and gain a section heading that will be a good target for the reference.